manifests/4.1/elasticsearch-operator.v4.1.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,257 @@
+  annotations:
+    categories: "OpenShift Optional, Logging & Tracing"
+    capabilities: "Seamless Upgrades"
+    certified: "false"
+    description: |-
+      The Elasticsearch Operator for OKD provides a means for configuring and managing an Elasticsearch cluster for tracing and cluster logging.
+      ## Prerequisites and Requirements
+      ### Elasticsearch Operator Namespace
+      The Elasticsearch Operator must be deployed to the global operator group namespace
+      ### Memory Considerations
+      Elasticsearch is a memory intensive application.  The initial
+      set of OKD nodes may not be large enough to support the Elasticsearch cluster.  Additional OKD nodes must be added
+      to the OKD cluster if you desire to run with the recommended(or better) memory. Each ES node can operate with a
+      lower memory setting though this is not recommended for production deployments.
+    containerImage: quay.io/openshift/origin-elasticsearch-operator:latest
+    createdAt: 2019-02-20T08:00:00Z
+    support: AOS Cluster Logging, Jaeger
+    alm-examples: |-
+        [
+            {
+                "apiVersion": "logging.openshift.io/v1",
+                "kind": "Elasticsearch",
+                "metadata": {
+                  "name": "elasticsearch"
+                },
+                "spec": {
+                  "managementState": "Managed",
+                  "nodeSpec": {
+                    "image": "quay.io/openshift/origin-logging-elasticsearch5:latest",
+                    "resources": {
+                      "limits": {
+                        "memory": "1Gi"
+                      },
+                      "requests": {
+                        "memory": "512Mi"
+                      }
+                    }
+                  },
+                  "nodes": [
+                    {
+                        "nodeCount": 1,
+                        "roles": ["client","data","master"],
+                        "redundancyPolicy": "SingleRedundancy"
+                    }
+                  ]
+                }
+            }
+        ]
+spec:
+  displayName: Elasticsearch Operator
+
+  description: |
+    The Elasticsearch Operator for OKD provides a means for configuring and managing an Elasticsearch cluster for use in tracing and cluster logging.
+    This operator only supports OKD Cluster Logging and Jaeger.  It is tightly coupled to each and is not currently capable of
+    being used as a general purpose manager of Elasticsearch clusters running on OKD.
+
+    It is recommended this operator be deployed to the **openshift-operators** namespace to properly support the Cluster Logging and Jaeger use cases.
+
+    Once installed, the operator provides the following features:
+    * **Create/Destroy**: Deploy an Elasticsearch cluster to the same namespace in which the Elasticsearch custom resource is created.
+
+  keywords: ['elasticsearch', 'jaeger']
+
+  replaces: elasticsearch-operator.v4.1.0
```

Comment:
@ecordell please confirm this is the correct syntax/format

Reply:
Looks correct to me - it should be the `metadata.name` of the previous release